source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-03-30 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+[Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) has been a great tool so far in providing a prepackaged development environment filled with production-grade, pre-configured tools, that makes the container developer's lives easier.
+
```

Comment:
The above paragraph is not required IMO. Also there is a disconnect between the paragraphs i.e. above and below. The below paragraph looks good to start the blog.